README.md

```diff
@@ -1,10 +1,74 @@
-# service-name
+# service-shortlink
 
-## Summary of the project
+| Branch | Status |
+|--------|-----------|
+| develop | ![Build Status]() |
+| master | ![Build Status]() |
```

Comment:
Missing build badge

Reply:
It's done, but it asks for an accepted review after the last changes ^^.